scalingo/resource_scalingo_addon.go

```diff
@@ -111,6 +131,28 @@ func resourceAddonRead(ctx context.Context, d *schema.ResourceData, meta interfa
 		return diag.Errorf("fail to store addon information: %v", err)
 	}
 
+	providers, err := client.AddonProvidersList(ctx)
+	if err != nil {
+		return diag.Errorf("fail to list addon providers: %v", err)
+	}
+	addonProvider := keepIf(providers, func(p *scalingo.AddonProvider) bool {
+		return p.ID == addon.AddonProvider.ID
+	})[0]
+	features := []string{}
+	if addonProvider.Category.Name == "Database" {
```

Comment:
In staging, this category is named `Databases` (https://admin.osc-st-fr1.st-sc.fr/platform_data/addons). Isn't there a more reliable way to get the databases addons? :/

Reply:
Not really :(

Reply:
ACK